modules/test/playwright/tests/batch-planner/import.spec.ts

```diff
@@ -1014,6 +1016,232 @@ test('can import CSV file with new and modified existing company scoped object e
+				indexedAsKeyword: false,
+				label: {
+					en_US: 'customAttachment',
+				},
+				name: 'diploma',
+				objectFieldSettings: [
+					{
+						name: 'acceptedFileExtensions',
+						value: 'jpeg, jpg, pdf, png',
+					} as any,
+					{
+						name: 'fileSource',
+						value: 'documentsAndMedia',
+					} as any,
+					{
+						name: 'maximumFileSize',
+						value: '100',
+					} as any,
+				],
+				required: false,
+				type: 'Long',
+			},
+		],
+		objectRelationships: [
+			{
+				deletionType: 'cascade',
+				externalReferenceCode: 'student-subjects-relationship',
+				label: {
+					en_US: 'Student subjects',
+				},
+				name: 'subjectStudents',
+				objectDefinitionExternalReferenceCode1: 'student-definition',
+				objectDefinitionExternalReferenceCode2: 'subject-definition',
+				objectDefinitionModifiable2: true,
+				objectDefinitionName2: 'Subject',
+				objectDefinitionSystem2: false,
+				objectField: {
+					DBType: 'Long',
+					businessType: 'Relationship',
+					externalReferenceCode:
+						'student-subjects-relationship-field',
+					indexed: true,
+					indexedAsKeyword: false,
+					indexedLanguageId: '',
+					label: {
+						en_US: 'Student subjects',
+					},
+					name: 'r_subjectStudents_c_studentId',
+					readOnly: 'false',
+					relationshipType: 'oneToMany',
+					required: true,
+					state: false,
+					system: false,
+					type: 'Long',
+					unique: false,
+				},
+				parameterObjectFieldId: 0,
+				parameterObjectFieldName: '',
+				reverse: false,
+				system: false,
+				type: 'oneToMany',
+			},
+		],
+		panelCategoryKey: 'control_panel.object',
+		pluralLabel: {
+			en_US: 'Students',
+		},
+		portlet: true,
+		restContextPath: '/o/c/students',
+		scope: 'company',
+		status: {
+			code: 0,
+		},
+	};
+
+	const {body: studentResponse} =
+		await objectDefinitionAPIClient.postObjectDefinition(
+			studentObjectDefinition
+		);
+
+	apiHelpers.data.push({id: studentResponse.id, type: 'objectDefinition'});
+
+	await apiHelpers.objectEntry.postObjectEntry(
+		{
+			diploma: {
+				fileBase64: 'R0lGODlhAQABAAAAACw=',
+				name: 'attachment.png',
+			},
+			name: 'Jane',
+			subject: {
+				name: 'Math',
+			},
+		},
+		'c/students'
+	);
+
+	let entries =
+		await apiHelpers.objectEntry.getObjectDefinitionObjectEntries(
+			'c/students'
+		);
+
+	const filePath = createTempFile(
+		getRandomString() + '.json',
+		`[{"diploma": {
+			"id":${entries.items[0].diploma.id},
+			"link":
+				{
+					"href": "${entries.items[0].diploma.link.href}",
+					"label": "${entries.items[0].diploma.link.label}"
+				},
+				"name": "${entries.items[0].diploma.name}"
+			},
+			"name": "John",
+			"r_subjectStudents_c_subjectERC": "Math"
+		}]`
+	);
+
+	await dataMigrationCenterPage.goto();
+	await dataMigrationCenterPage.goToImportFile();
+
+	await dataMigrationCenterPage.importFile(
+		'com.liferay.object.rest.dto.v1_0.ObjectEntry#C_Student',
+		filePath,
+		'INSERT',
+		'PARTIAL_UPDATE'
+	);
+
+	entries =
```

Comment:
In my local tries this has always had 1 entry. The issue for me is that the import process doesn't start until this line. Is this happening also to you?